test/helpers/pagelets_helper_test.rb

```diff
@@ -1,17 +1,17 @@
 require 'test_helper'
+require 'statics_isolator_helper'
 
 class PageletsHelperTest < ActionView::TestCase
+  include Isolator
   include PageletsHelper
 
+  isolate_config Pagelets::Manager
```

Comment:
I'm not sure I understand the need for this whole module. Why not just define what you want to be defined for the specific text context in the setup block and clear it in the tear down block?